model/lib/status-log.js

```diff
@@ -13,6 +13,15 @@ module.exports = memoize(function (db) {
 		time: { type: db.DateTime, required: true },
 		official: { type: User },
 		officialFullName: { type: StringLine },
-		text: { type: db.String, required: true }
+		text: { type: db.String, required: true },
+		toJSON: { value: function (ignore) {
+			return {
+				label: this.getOwnDescriptor('label').valueToJSON(),
+				time: this.getOwnDescriptor('time').valueToJSON(),
+				officialFullName: this.officialFullName
+					&& this.getOwnDescriptor('officialFullName').valueToJSON(),
```

Comment:
This doesn't change anything, still we end with `officialFullName: undefined` which we don't want